package.json

```diff
@@ -12,24 +12,21 @@
     "morgan": "^1.9.0"
   },
   "scripts": {
-    "build": "npm run clean && npm-run-all --parallel build:static build:node",
-    "build:node": "./tasks/build-node-scripts.sh",
-    "build:prod": "NODE_ENV=production npm run build",
-    "build:static": "./tasks/build-static.sh",
     "clean": "rm -rf dist",
-    "lint": "npm-run-all --parallel lint:eslint lint:stylelint",
-    "lint:eslint": "eslint 'src/**/*.js'",
+    "lint:eslint": "eslint src",
     "lint:stylelint": "stylelint 'src/**/*.css'",
-    "serve": "npm run build && npm-run-all --parallel watch server",
-    "start": "node dist/server/server.js",
-    "server": "node ./node_modules/.bin/nodemon node dist/server/server.js",
-    "test": "npm run test:node && npm run test:static",
-    "test:node": "./tasks/test-node.sh",
-    "test:static": "./node_modules/.bin/karma start",
-    "tools:import_products": "./node_modules/.bin/babel-node tools/import_products.js",
-    "watch": "npm-run-all --parallel watch:static watch:node",
-    "watch:node": "./node_modules/.bin/watch 'npm run build:node' src/server src/data",
-    "watch:static": "./node_modules/.bin/watch 'npm run build:static' src/client"
+    "lint": "npm-run-all --continue-on-error lint:eslint lint:stylelint",
+    "build:frontend": "./tasks/build-static.sh",
+    "build:backend": "./tasks/build-node-scripts.sh",
+    "build": "npm-run-all clean build:backend build:frontend",
+    "build:prod": "NODE_ENV=production npm run build",
+    "nodemon:frontend": "nodemon --watch src/client --exec 'npm run build:frontend'",
+    "nodemon:backend:serve": "nodemon --watch src/data --watch src/server --watch src/services --watch src/templates --exec 'npm-run-all build:backend start'",
```

Comment:
@devnook I've tried a few different configurations here and this still ends up being the simplest. If I separate the backend build and server restart that ends up with another target that needs to watch the `dist` directory with a little delay to ensure the server *only* restarts when the build is done. I think coupling them here is fine, because they are sequential and depend on each other.

Ideally, a developer would never run or care about either of the `nodemon:`* targets individually. Developers should only have to use the `start:dev` command.